taskcluster/ac_taskgraph/loader/build_config.py

```diff
@@ -27,7 +27,16 @@
     "0000000000000000000000000000000000000000000000000000000000000000", # sha256
 )
 
-CONFIGURATIONS_WITH_DEPENDENCIES = ("implementation", "api")
+CONFIGURATIONS_WITH_DEPENDENCIES = (
+    "api",
+    "compileOnly",
```

Comment:
This makes sense to me after reading https://github.com/mozilla-mobile/android-components/issues/8540#issuecomment-700942877.